src/DoctrineORMModule/Service/ConfigurationFactory.php

```diff
@@ -152,6 +152,10 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
             $config->setSecondLevelCacheEnabled();
             $config->setSecondLevelCacheConfiguration($cacheConfiguration);
         }
+        
+        if($schemaFilter = $options->getSchemaFilter()) {
```

Comment:
p.s. I'm a proponent of constant diction in an application.  So 
```
$config->setFilterSchemaAssetsExpression($schemaFilter);
```
breaks constant diction because the same thing is being called two things here.